browser/brave_tab_helpers.cc

```diff
@@ -107,8 +107,8 @@
 namespace brave {
 
 void AttachTabHelpers(content::WebContents* web_contents) {
-  brave_shields::BraveShieldsWebContentsObserver::CreateForWebContents(
-      web_contents);
+  AttachRequiredTabHelpers(web_contents);
```

Comment:
I know we are adding the comment to the header file, but I think we should add a comment here too, since this is where a developer would think about adding a new tab helper, and would be easy to miss during code review.